daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Multi-app Run (MapR)"
```

Comment:
MapR sounds like a proprietary software! 😢 There is also an IBM software https://www.ibm.com/docs/en/spss-analytic-server/3.1.1?topic=configuration-mapr-overview 

I propose that we discuss further before merging this.